nautilus_trader/data/engine.pyx

```diff
@@ -1547,19 +1543,6 @@ cdef class DataEngine(Component):
         self._log.warning(f"Cannot handle request: no client registered for '{request.client_id}', {request}")
 
     cpdef void _query_catalog(self, RequestData request):
-        """
```

Comment:
Small point on code style - we typically don't use docstrings for documenting private methods, as they are meant for public facing docs. Any docs for these should be regular comments.